package.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "bourbon-neat",
+  "name": "just-neat",
```

Comment:
This is actually necessary because there is already a npm package called `neat`.